app/services/twilio_service.rb

```diff
@@ -37,20 +44,11 @@ def twilio_client
     )
   end
 
-  def send_sms(params = {})
-    params = params.reverse_merge(from: from_number)
-    @client.messages.create(params)
-  end
-
   def from_number
     "+1#{account['number']}"
   end
 
-  def self.accounts
-    Rails.application.secrets.twilio_accounts
-  end
-
-  def self.random_account
-    accounts.sample
+  def random_account
+    JSON.parse(Figaro.env.twilio_accounts).sample
```

Comment:
Edit: I see what you mean. What we could do to avoid parsing every time is to set the accounts array as a constant, then call `.sample` on the constant.